app/logic/emailHandler.py

```diff
@@ -92,7 +102,7 @@ def retrieve_recipients(self, recipients_category):
             recipients = (User.select()
                 .join(Interest)
                 .join(Program, on=(Program.id==Interest.program))
-                .where(Program.id( self.program_id)))
+                .where(Interest.program_id == self.program))
```

Comment:
In our peewee queries we try to avoid using `_id` unless we really need to. For example, even though peewee lets us compare ids and objects here, let's use `Interest.program`.

Reply:
dine